examples/json_validation_tests/submission_valid-1.json

```diff
@@ -0,0 +1,48 @@
+{
+    "object_id": {
+        "alias" : "submission_XF40", 
+        "center_name" : "EBI-TEST"
+    },
+    "object_title": "Submission XF40",
+    "object_description": "EBI submission project XF40 of 1000 samples and its 1500 sequencing runs",
+    "additional_collaborators": [
+        {
+            "collaborator_rights": "read_and_write",
+            "collaborator_contact_details": {
+                "institution_name": "European Bioinformatics Institute (EBI)",
+                "email_address": "example@gmail.com"
+            }
+        },
+        {
+            "collaborator_rights": "read_only",
+            "collaborator_contact_details": {
+                "individual_full_name": "Isaac Asimov",
+                "institution_name": "Boston University",
+                "email_address": "example@gmail.com"
```

Comment:
I suggest not using gmail as an example since we do not accept gmail email accounts for users.

Reply:
True